Examples/UsingVirtualListExample.py

```diff
@@ -179,13 +178,13 @@ def HandleSort(self, evt):
         self.reorderList = [x[0] for x in cur.fetchall()]
         self.myOlv.RefreshObjects()
 
-        print "Sorting took %2f seconds." % (time.clock() - start)
+        print("Sorting took %2f seconds." % (time.perf_counter() - start))
 
 
 if __name__ == '__main__':
-    app = wx.PySimpleApp(0)
+    print('Using {} ({}) from {}.'.format(OLV.__name__, OLV.__version__, OLV.__path__))
```

Comment:
This would also be a good place for an f-string

Reply:
See https://github.com/driscollis/ObjectListView3/pull/20#discussion_r1685029961